automatic/vcredist2017/update.ps1

```diff
@@ -0,0 +1,27 @@
+Import-Module AU
+Import-Module "$env:ChocolateyInstall\helpers\chocolateyInstaller.psm1"
+Import-Module "$PSScriptRoot\..\..\scripts/au_extensions.psm1"
+
+$x64Release = 'https://aka.ms/vs/15/release/VC_redist.x64.exe'
+
+function global:au_SearchReplace {
+  @{
+    ".\vcredist2017.nuspec" = @{
+      "(\<dependency .+? version=)`"([^`"]+)`"" = "`$1`"[$($Latest.Version)]`""
```

Comment:
I'm having seconds thoughts about restricting this to an exact version.
I think perhaps having a minimum version would be better in this case.

Consider this:

In the future, perhaps vcredist2019 (or later) is released, and is also backwards
compatible with vcredist2015 and vcredist2017.
If we use exact constraint, users wouldn't be able to install the vcredist2019 package
if they already have vcredist2017, unless they first uninstall that package.
Thoughts?

Reply:
I think this feeds into future plans for the Chocolatey nuspec, namely:

```
    <!--<provides>NOT YET IMPLEMENTED</provides>-->
    <!--<conflicts>NOT YET IMPLEMENTED</conflicts>-->
    <!--<replaces>NOT YET IMPLEMENTED</replaces>-->
```

So I don't think this is something to worry about here, at this point in time.

Reply:
> So I don't think this is something to worry about here, at this point in time.

I'm not so sure, that would depend on what comes first.
Will vcredist2019 (or something along the lines) come before choco adds that support or not (of course, only time will tell here).